src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/query-time-range.mdx

```diff
@@ -0,0 +1,2701 @@
+---
+title: Setting a query's time range
+tags:
+  - Query your data
+  - 'NRQL: New Relic Query Language'
+  - Get started
+translate:
+  - jp
+  - kr
+metaDescription: "A detailed description of how to set the timerange of a NRQL query."
+freshnessValidatedDate: never
+---
+
+import exampleFacetedLeftJoin from 'images/nrql_screenshot-crop_example-faceted-left-join.webp'
+
+import exampleFacetedInnerJoin from 'images/nrql_screenshot-crop_example-faceted-inner-join.webp'
+
+import exampleColumnarInnerJoin from 'images/nrql_screenshot-crop_example-columnar-inner-join.webp'
+
+import exampleJoinedPercentile from 'images/nrql_screenshot-crop_example-joined-percentile.webp'
+
+import exampleJoinedMedian from 'images/nrql_screenshot-crop_example-joined-median.webp'
+
+import queriesnrqlApdexNRQLQueryBuilder from 'images/queries-nrql_screenshot-full_apdex-NRQL-query-builder.webp'
+
+import lowerExample from 'images/lowerExample.webp'
+
+import percentile0 from 'images/percentile_0.webp'
+
+import positionExample from 'images/positionExample.webp'
+
+import queriesnrqlPercentileNRQLQueryBuilder from 'images/queries-nrql_screenshot-full_percentile-NRQL-query-builder.webp'
+
+import queriesnrqlFilterNRQLQueryBuilder from 'images/queries-nrql_screenshot-full_filter-NRQL-query-builder.webp'
+
+import screenshotInsightsFilter00 from 'images/screenshot_insights_filter_0_0.webp'
+
+import substringExample from 'images/substringExample.webp'
+
+import upperExample from 'images/upperExample.webp'
+
+[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language) was
+created to query telemetry data.
+
+Every query that is executed only considers telemetry data that falls in a specified time range. If a query request
+does not specify a time range, the last hour of telemetry data will be queried.
+
+## Specifying a time range [#spec-time-range]
+There are two ways to specify the time range:
+
+### Using the dashboard [time picker](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/#dash-time-picker)
+
+When viewing a dashboard, the time picker time range will supersede a query's `SINCE` and `UNTIL` clauses, unless the widgets has been configured to disable
```

Comment:
unless the dashboard widgets have

Reply:
Since abbreviation support is limited (due to non-unique abbreviations across all timezones) I'm opting to remove abbreviations references from the docs, rather than sort out the Casey Time issue. Especially since there may be other abbreviations that are missed in the generated report.